<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KNewWalletDialogGpg</class>
 <widget class="QWidget" name="KNewWalletDialogGpg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please select the signing key from the list below:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="listCertificates">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>E-Mail</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Key-ID</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listCertificates</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>KNewWalletDialogGpg</receiver>
   <slot>onItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onItemSelectionChanged()</slot>
 </slots>
</ui>
